backend/app/Http/Controllers/Api/VisitController.php

```diff
@@ -58,4 +62,11 @@ public function getPageViewsCountForFilterData(GetPageViewsCountFilterHttpReques
         $response = $this->getPageViewsCountAction->execute(GetPageViewsCountRequest::fromRequest($request));
         return ApiResponse::success(new ButtonResource($response));
     }
+
+    public function createVisit(CreateVisitHttpRequest $request): ApiResponse
+    {
+        $response = $this->createVisitAction->execute(CreateVisitRequest::fromRequest($request));
+
+        return ApiResponse::success(new VisitResource($response->visit()));
```

Comment:
as it is not internal Api, it shouldn't return any data in response, status 200 is enough. And probably it should be in another namespace. Like OpenApi, or something